src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -71,21 +71,24 @@ public Errors validateFull(OverseasEntitySubmissionDto overseasEntitySubmissionD
     }
 
     private void validateFullUpdateDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
```

Comment:
Are any changes needed for partial updates? Although I don't think the `validatePartialUpdateDetails` method is used

Reply:
Discussed, this will be moved to another ticket